ando/tools/generator/AnDOGenerator.py

```diff
@@ -209,7 +209,7 @@ def generate_data_files(self, mode='link'):
                 if len(files) > 1:
                     split = f'_split-{i}'
 
-                new_filename = filename_stem + key + split + suffix + postfix
+                new_filename = filename_stem + key + split + postfix + suffix
```

Comment:
so I just forgot to add the postfix in this line... That was not very clever of me :/ Thanks for fixing it!

Reply:
=D It's ok, I've also done some things not clever in AnDO ! Glad It helps \o/